app/controllers/LeaveFeedbackController.scala

```diff
@@ -18,36 +18,39 @@ package controllers
 
 import audit.Auditable
 import config.AtedFrontendAuditConnector
-import controllers.auth.{AtedFrontendAuthHelpers, AtedRegime, AtedSubscriptionNotNeededRegime}
+import controllers.auth.AuthAction
 import models.LeaveFeedback
+import play.api.Play.current
 import play.api._
+import play.api.i18n.Messages.Implicits._
 import play.api.mvc._
+import services.DelegationService
+import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.audit.model.{Audit, EventTypes}
 import uk.gov.hmrc.play.config.AppName
-import play.api.i18n.Messages.Implicits._
-import play.api.Play.current
 
 import scala.concurrent.Future
-import uk.gov.hmrc.http.HeaderCarrier
 
 
 
-trait LeaveFeedbackController extends AtedBaseController with AtedFrontendAuthHelpers with Auditable {
+trait LeaveFeedbackController extends AtedBaseController with Auditable with AuthAction {
 
-  def view(returnUri: String) = AuthAction(AtedSubscriptionNotNeededRegime) {
-    implicit atedContext =>
+  def view(returnUri: String): Action[AnyContent] = Action.async { implicit request =>
+    authorisedAction { implicit authContext =>
```

Comment:
RED - This needs to be the less strict authAction.

Reply:
authorisedForNoEnrolments*

Reply:
Replaced with authorisedForNoEnrolments